test/helper.rb

```diff
@@ -149,6 +149,11 @@ def md.directory?
       # Test DH parameters, 2048 bit long safe prime, generator 2 for dh_params in PEM format
       'dh_params.dh_pem' => mockfile.call('dh_params.dh_pem'),
       'default.toml' => mockfile.call('default.toml'),
+      '/etc/postgresql/9.5/main/pg_ident.conf' => mockfile.call('pg_ident.conf'),
+      'C:/etc/postgresql/9.5/main/pg_ident.conf' => mockfile.call('pg_ident.conf'),
+      '/etc/postgresql/9.5/main' => mockfile.call('9.5.main'),
```

Comment:
This is likely because of your direct call to `inspec.postgres.conf_dir` in the `initialize` method. Unit tests should not have to rely on OS or filesystem status as it's about testing the code/messages rather than a functional test.

You'll need to add a mock in your unit tests to cover this as adding these directory mockloader calls isn't appropriate.

It will look something like this (writing this off-the-cuff, haven't tested this):

```ruby
describe 'blah' do
  let(:resource) { load_resource... }
  let(:inspec) { mock }
  let(:postgres) { mock }

  before do
    resource.stubs(:inspec).returns(inspec)
    inspec.stubs(:postgres).returns(postgres)
    postgres.stubs(:conf_dir).returns('/test/path/to/postgres')
  end

  it 'does something' do
    entries = resource.where ...
  end
end
```

This should stub/mock the call to `inspec.postgres.conf_dir` so that it always returns the string `/test/path/to/postgres` so it works regardless of operating system and whether or not PostgreSQL is installed on the machine performing the unit tests. And then I'd change the helper config to mock out `/test/path/to/postgres/pg_ident.conf` instead of the one in `/etc/postgres...` so that you're 100% certain your testing just your code, not some integration point on the system itself.

Search the tests in `test/unit` for additional examples of how to use `mock` and `stubs`. Let me know if you get hung up on this, but we have to square this away before this can be merged.